vsts/pipelines/PlatformBinaries/dotnetcore.yml

```diff
@@ -7,7 +7,7 @@ stages:
   - job: DotNetCore_Buster
     timeoutInMinutes: 250
     pool:
-      name: OryxLinux
+      vmImage: 'ubuntu-latest'
```

Comment:
What was the reason that we want to replace this pool image name?

Reply:
we dont want to manage this private buildagent pool "oyxlinux" any more for better and secured supply chain